.github/renovate.json

```diff
@@ -1,26 +1,15 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
+  "globalExtends": [
     "config:base"
   ],
+  "extends": ["group:allNonMajor"],
   "packageRules": [
     {
       "description": "Create a PR whenever there is a new major version",
       "matchUpdateTypes": [
         "major"
       ]
-    },
-    {
-      "description": "Create a PR grouping all non-major dependencies",
-      "matchPackagePatterns": [
-        "*"
-      ],
-      "matchUpdateTypes": [
-        "minor",
-        "patch"
-      ],
-      "groupName": "all non-major dependencies",
-      "groupSlug": "all-minor-patch"
     }
   ],
   "ignorePaths": [
```

Comment:
Could you also make the schedule more frequent, like once a month, now that the PRs are going to be consolidated"

Reply:
Sounds good, changed config to extend the schedule:monthly renovate preset